spec/behavior_spec.rb

```diff
@@ -623,6 +623,22 @@
     expect(described_class.beautify(source)).to eq(expected)
   end
 
+  it "does not break for > within an attribute value" do
```

Comment:
This verifies it doesn't break code which is already formatted correctly, but there should probably be some other tests.